File: OCAirLines/OCAirLines.Database/Models/Pesquisa.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OCAirLines.Database.Models
{
    public class Pesquisa
    {
        public int Id { get; set; }
        public int UsuarioId { get; set; }
        public int EmpresaId { get; set; }
        public string Empresa { get; set; }
        public int OrigemId { get; set; }
        public string Origem { get; set; }
        public int DestinoId { get; set; }
        public string Destino { get; set; }
        public decimal Preco { get; set; }
        public DateTime DtSaida { get; set; }
        public DateTime DtChegada { get; set; }
        public bool Status { get; set; }

        public Usuario Usuario { get; set; }
    }
}

File: OCAirLinesFront/src/app/core/models/webapi/pesquisa.ts
export interface Pesquisa {
  id: number;
  usuarioId: number;
  empresaId: number;
  empresa: string;
  origemId: number;
  origem: string;
  destinoId: number;
  destino: string;
  preco: number;
  dtSaida: string;
  dtChegada: string;
}
